src/useIdle.ts

```diff
@@ -46,7 +46,7 @@ const useIdle = (ms: number = oneMinute, initialState: boolean = false, events:
       }
       off(document, 'visibilitychange', onVisibility);
     };
-  }, events);
+  }, [ms, events]);
```

Comment:
I guess, it should be

```js
[ms, ...events]
```

Reply:
@streamich
I think it should have been `[events]` in the first place.
Because here `events` is the actual dependency, not items in it.

In fact, if the array updates with less or more events like this:
Before: `['click', 'mousemove']`
After: `['click']`
useEffect won't update and throw a warning saying `The final argument passed to useEffect changed size between renders`.

See this example
https://codesandbox.io/s/n3ow36wql

Reply:
Thanks, good to know.